ui/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureextensionui",
     "author": "Microsoft Corporation",
-    "version": "0.29.0",
+    "version": "0.30.0",
```

Comment:
tl;dr please update to 0.29.1 instead. Also update the version in package-lock.json

More explanation:
Normally, you're [supposed to bump](https://semver.org/) the major version for breaking changes. Since we're not ready to commit to "1.0.0" yet, we shift the meaning of each position down. Thus we bump the minor version for breaking changes and the patch version for anything else. Since you did not introduce any breaking changes, you don't need to bump the minor version.

Reply:
Thanks! Good to know.